int8 ID
---
int8 Power
